### PR TITLE
Update Where Are My Certs section.

### PR DESCRIPTION
**Pull request by jsha**
This combines the `cert.pem` and `chain.pem` sections into a single paragraph,
making it clearer that they are closely connected. It also adds text indicating
that they are less common and moves them below the section for `fullchain.pem`.

Also update Getting Help to emphasize forums, and fix Nginx capitalization.

### CODE REVIEW

File: docs/using.rst [hunk 1]
```diff
@@ -503,40 +503,40 @@ The following files are available:
...
-  This is what Apache < 2.4.8 needs for `SSLCertificateChainFile
-  <https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile>`_,
-  and what nginx >= 1.3.7 needs for `ssl_trusted_certificate
-  <http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_trusted_certificate>`_.
-
 ``fullchain.pem``
   All certificates, **including** server certificate. This is
   concatenation of ``cert.pem`` and ``chain.pem``.
 
   This is what Apache >= 2.4.8 needs for `SSLCertificateFile
```
Comment (pde) [inline docs/using.rst]: Not strictly "needs", maybe "This is the best thing to use in Apache >= 2.4.8 for..."

File: docs/using.rst [hunk 2]
```diff
@@ -503,40 +503,40 @@ The following files are available:
...
-  This is what Apache < 2.4.8 needs for `SSLCertificateFile
-  <https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatefile>`_.
-
-``chain.pem``
-  All certificates that need to be served by the browser **excluding**
-  server certificate, i.e. root and intermediate certificates only.
-
-  This is what Apache < 2.4.8 needs for `SSLCertificateChainFile
-  <https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile>`_,
-  and what nginx >= 1.3.7 needs for `ssl_trusted_certificate
```
Comment (pde) [inline docs/using.rst]: A question this leaves me with: does nginx >= 1.3.7 need *both* `ssl_trusted_certificate` and `ssl_certificate`?

File: docs/using.rst [hunk 3]
```diff
@@ -503,40 +503,40 @@ The following files are available:
...
+  Apache < 2.4.8 needs these for `SSLCertificateFile
+  <https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatefile>`_.
+  and `SSLCertificateChainFile
+  <https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile>`_,
+  respectively.
 
-Please note, that **you must use** either ``chain.pem`` or
-``fullchain.pem``. In case of webservers, using only ``cert.pem``,
-will cause nasty errors served through the browsers!
+  If you're using OCSP stapling with Nginx >= 1.3.7, ``chain.pem`` should be
```
Comment (pde) [inline docs/using.rst]: Oh, maybe this was the answer to my previous question ;).  I guess Nginx >= 1.3.7 "needs" `ssl_trusted_certificate`, but only with OCSP stapling?

**Comment by pde** [comment]
Otherwise LGTM.

File: docs/using.rst [hunk 4]
```diff
@@ -503,40 +503,40 @@ The following files are available:
...
-  and what nginx needs for `ssl_certificate
+  and what Nginx needs for `ssl_certificate
   <http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_certificate>`_.
 
+``cert.pem`` and ``chain.pem`` (less common)
+  ``cert.pem`` contains the server certificate by itself, and
+  ``chain.pem`` contains the additional intermediate certificate or
+  certificates that web browsers will need in order to validate the
+  server certificate. If you provide one of these files to your web
+  server, you **must** provide both of them, or browsers will show
```
Comment (ohemorange) [inline docs/using.rst]: "or else some browsers will mysteriously show "xXX" errors some of the time"

Comment (ohemorange) [inline docs/using.rst]: + link to some page describing the problem

Comment (ohemorange) [inline docs/using.rst]: + = +

File: docs/using.rst (hunk 3 above)
Comment (jsha) [inline docs/using.rst]: Yep

File: docs/using.rst (hunk 2 above)
Comment (pde) [inline docs/using.rst]: maybe add that parenthetical, "(if you use OCSP stapling)".

File: docs/using.rst [hunk 5]
```diff
@@ -507,8 +507,9 @@ The following files are available:
   <http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_certificate_key>`_.
 
 ``fullchain.pem``
-  All certificates, **including** server certificate. This is
-  concatenation of ``cert.pem`` and ``chain.pem``.
```
Comment (pde) [inline docs/using.rst]: I think the concatenation statement was correct, right? Did you remove it because you thought it was confusing?

**Comment by zeropopular** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  mrniceguyzero@gmail.com
    Domain thezeroworld.com has exceeded the max emails per hour (8/7 (114%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o4.sgmail.github.com ([192.254.112.99]:37162)
	by node87.certushost.com with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.87)
	(envelope-from <bounces+848413-7f84-zero=thezeroworld.com@sgmail.github.com>)
	id 1btkIy-00077t-5m
	for zero@thezeroworld.com; Mon, 10 Oct 2016 19:48:24 -0400
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=JJ4Oi4BK/F5iqjup4uBSwphxlQo=; b=hYozIm/HeQSWUu9k
	sQjQ9QGh4HZtSXfXbT3zFkZqURltapTWtB/ScQ15I/M/I1fnrAOhuiHAEltMbeDI
	dBqDb43tEj8bLKsn13dO1nFTytwP5FTNj+krH3N2ClHVo7IHEVh4XUeUxSAg/nh9
	hEiT9RFseOtguXnfWwccKv7xldw=
Received: by filter0974p1mdw1.sendgrid.net with SMTP id filter0974p1mdw1.22481.57FC289320
        2016-10-10 23:47:31.441517855 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0005p1iad1.sendgrid.net (SG) with ESMTP id PpKl6F69Q2uf_yTkUz13NQ
	for <zero@thezeroworld.com>; Mon, 10 Oct 2016 23:47:31.331 +0000 (UTC)
Date: Mon, 10 Oct 2016 16:47:31 -0700

File: docs/using.rst (hunk 5 above)
Comment (jsha) [inline docs/using.rst]: Yeah, this line was confusing with the reordering, and also unnecessary. If you feel like it's important information, I could add it below in the `cert.pem` / `chain.pem` section, something like "If you concatenate cert.pem and chain.pem, you would identical contents as fullchain.pem." But I'm not sure that's useful to anyone.

**Pull request merged**